data/data/azure/bootstrap/main.tf

```diff
@@ -81,6 +82,20 @@ data "ignition_config" "redirect" {
   }
 }
 
+resource "azurerm_public_ip" "bootstrap_public_ip" {
+  sku                 = "Standard"
+  location            = var.region
+  name                = "${var.cluster_id}-bootstrap-pip"
+  resource_group_name = var.resource_group_name
+  allocation_method   = "Static"
+  domain_name_label   = var.dns_label
```

Comment:
[`domain_name_label`](https://www.terraform.io/docs/providers/azurerm/r/public_ip.html#domain_name_label) is optional, and I don't think we need a dns name registered for the public ip address for bootstrap. Let's drop this.

Reply:
Fixed.

Reply:
bf5a376 +1